lib/manageiq_performance/browser_mode_middleware.rb

```diff
@@ -29,7 +29,16 @@ def enable_performance_middleware? env
     end
 
     def set_performance_middleware_proc
-      if ManageIQPerformance.config.browser_mode.always_on?
+      if ManageIQPerformance.config.browser_mode.whitelist?
```

Comment:
I expected `always_on?` to be higher precedence than `'whitelist'

Reply:
@kbrock I didn't really give it much though, but I can flip if you think that makes sense.

I think the only reason I did it the way I did was because I was adding a lot of these features on a single branch, and I just wanted to keep the `whitelist` to be the ting I was commenting and commenting in the config.  But that isn't necessarily a good idea to make it intuitive, so I will change.